crates/common/tedge_config/src/lib.rs

```diff
@@ -1,14 +1,17 @@
 mod sudo;
-pub mod tedge_toml;
 pub use sudo::SudoCommandBuilder;
 pub mod cli;
 mod system_toml;
-
 pub use self::system_toml::*;
+
+pub mod tedge_toml;
 pub use self::tedge_toml::error::*;
-pub use self::tedge_toml::models::*;
-pub use self::tedge_toml::tedge_config::*;
 pub use self::tedge_toml::tedge_config_location::*;
+pub use tedge_toml::models;
+pub use tedge_toml::tedge_config::TEdgeConfig;
+pub use tedge_toml::tedge_config::TEdgeConfigDto;
+pub use tedge_toml::tedge_config::TEdgeConfigReader;
```

Comment:
```suggestion
pub use self::tedge_toml::models;
pub use self::tedge_toml::tedge_config::TEdgeConfig;
pub use self::tedge_toml::tedge_config::TEdgeConfigDto;
pub use self::tedge_toml::tedge_config::TEdgeConfigReader;
```

or remove `self::` from the others, it's just nice to be consistent.